src/app/api/getPosts.ts

```diff
@@ -0,0 +1,65 @@
+import { DateTime } from 'luxon';
+import { Octokit } from 'octokit';
+import slugify from 'slugify';
+
+function toSlug(str: string) {
+  return slugify(str).toLowerCase();
+}
+
+export type Post = {
+  slug: string,
+  date: string,
+  title: string,
+  summary: string,
+  tags: string[],
+  content: string,
+};
+
+export async function getPosts(): Promise<Post[]> {
+  const octokit = new Octokit();
+  const issues = await octokit.rest.issues.listForRepo({
+    owner: 'maffin-io',
+    repo: 'maffin-blog',
+    creator: 'argaen',
```

Comment:
This is so no random issues appear as blog posts in the website. The good thing is that new authors can start creating posts and when we add them in the list their posts will appear!